autoscheduler/scraper/tests/api_tests.py

```diff
@@ -453,6 +460,21 @@ def test_api_course_search_gives_correct_results_csce_space_310_lower(self):
         self.assertEqual(response.status_code, 200)
         self.assertEqual(response.json(), expected)
 
+    def test_api_course_search_gives_correct_results_by_title(self):
+        """ Tests that /api/course/search gives the correct response for a search
+            by title
+        """
+        # Arrange
+        expected = {'results': ['CSCE 310 - DATABASE SYSTEMS']}
+        data = {'search': 'Database S', 'term': '201731'}
+
+        # Act
+        response = self.client.get('/api/course/search', data=data)
```

Comment:
Does this send data as query params or as request body?

Reply:
Parameters, [here's the documentation for it](https://docs.djangoproject.com/en/3.0/topics/testing/tools/#django.test.Client.get)